projects/zstd-safe/build.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash -eu
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+# Retrieve zstd submodule
+git submodule init
+git submodule update
+
+# Build the fuzzers and project source code
+cd zstd-safe
```

Comment:
```suggestion
cd zstd-safe/fuzz
```

Reply:
Thanks for the suggestion. But the Cargo Fuzz default path is the fuzz directory, I don't think this change is necessary.

Reply:
Ah, sorry for the noise then! Not familiar with cargo-fuzz yet.

Reply:
No problem. Thanks for your suggestion.